lib/improver/tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -336,10 +353,10 @@ def test_list_of_constraints(self):
                       AuxCoord(0.03, units='mm hr-1')]
         result = plugin.construct_extract_constraint(diagnostics,
                                                      thresholds)
-        expected = ("Constraint(name='probability_of_lwe_snowfall_rate',"
-                    " coord_values={'threshold': 0.03})")
+
+        expected = ("iris.Constraint(name='probability_of_lwe_snowfall_rate', "
+                    "threshold=lambda cell: 0.03*0.99 < cell < 0.03*1.01)")
         self.assertIsInstance(result, list)
-        self.assertIsInstance(result[0], iris.Constraint)
```

Comment:
Similar we should test this is returning a list of strings and replace str(result[1]) with result[1]

Reply:
:white_check_mark: